src/meta/src/model/stream.rs

```diff
@@ -564,6 +566,25 @@ impl TableFragments {
         }
     }
 
+    /// Retrieve the internal tables map of the whole graph.
+    ///
+    /// See also [`crate::stream::StreamFragmentGraph::incomplete_internal_tables`].
+    pub fn internal_tables(&self) -> BTreeMap<u32, Table> {
```

Comment:
I suppose these are "completed" internal tables, in contrast to the incomplete_internal_tables below. It would be good to mention that in the docstring.